CONTRIBUTING.md

```diff
@@ -51,6 +51,65 @@ implementation of the issue, ask them in the issue instead of the PR.
 
 The following instructions are for developers and contributors to cuDF OSS development. These instructions are tested on Linux Ubuntu 16.04 & 18.04. Use these instructions to build cuDF from source and contribute to its development.  Other operating systems may be compatible, but are not currently tested.
 
+### Code Formatting
+
+#### Python
+
+cuDF uses [Black](https://black.readthedocs.io/en/stable/),
+[isort](https://readthedocs.org/projects/isort/), and
+[flake8](http://flake8.pycqa.org/en/latest/) to ensure a consistent code format
+throughout the project. `Black`, `isort`, and `flake8` can be installed with
+`conda` or `pip`:
+
+```bash
+conda install black isort flake8
+```
+
+```bash
+pip install black isort flake8
+```
+
+These tools are used to auto-format the Python code, as well as check the Cython
+code in the repository. Additionally, there is a CI check in place to enforce
+that committed code follows our standards. You can use the tools to
+automatically format your python code by running:
+
+```bash
+isort --recursive --atomic --apply python
```

Comment:
Are these already part of the PR template/checklist?

Might be good to suggest that these tools be run from $CUDF_PROJECT_DIR/python to make sure all files are checked.

Reply:
It's run from `$CUDF_PROJECT_DIR` but I'll add it. Didn't look at the template/checklist but I'll add them.